Homework5/BookCatalog/Catalog.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BookCatalog
+{
+    class Catalog
+    {
+        private List<Book> CatalogOfBooks = new List<Book>();
+
+        public void AddBook(Book book)
+        {
+            if (book is null)
+            {
+                throw new ArgumentNullException();
+            }
+
+            CatalogOfBooks.Add(book);
```

Comment:
Lets ensure that if the book ISBN is already added, then the record is updated. 

Current implementation allows to add a few books with the same ISBN, but only the first one may be retrieved by GetBook